ros_controllers/package.xml

```diff
@@ -17,17 +19,17 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <run_depend>imu_sensor_controller</run_depend>
-  <run_depend>force_torque_sensor_controller</run_depend>
-  <run_depend>position_controllers</run_depend>
-  <run_depend>velocity_controllers</run_depend>
-  <run_depend>effort_controllers</run_depend>
-  <run_depend>forward_command_controller</run_depend>
-  <run_depend>joint_state_controller</run_depend>
-  <run_depend>joint_trajectory_controller</run_depend>
-  <run_depend>diff_drive_controller</run_depend>
-  <run_depend>gripper_action_controller</run_depend>
-  <run_depend>rqt_joint_trajectory_controller</run_depend>
+  <exec_depend>imu_sensor_controller</exec_depend>
```

Comment:
alphabetize (well, this is a general suggestion which can be done on another PR)